### ISSUE TITLE
cannot reproduce result in your paper with InceptionV4

### ISSUE THREAD

**Issue by tuyunbin** [ramakanth-pasunuru/video_captioning_rl#9]
Hi, I tried to use InceptionV4 as CNN encoder to train the Baseline-XE model, but I cannot get the results like your paper. When finishing trarining, I got three .pth, i.e., epoch1, epoch2, and epoch3. I didn't konw which is more better, so I tested them respectively. I found epoch3.pth is the best among three. But the results are still lower than your paper.
I got the results as follows:
2020-04-22 18:00:49,045:INFO::Results:
2020-04-22 18:00:49,046:INFO::BLEU-4:0.3745473407866488
2020-04-22 18:00:49,046:INFO::METEOR:0.26116567752846037
2020-04-22 18:00:49,046:INFO::ROUGE_L:0.5878605132246427
2020-04-22 18:00:49,046:INFO::CIDEr:0.4295072766573666
2020-04-22 18:00:49,046:INFO::AVG:0.4132702020492796,
which are much lower than your paper, B4 38.6,  M 27.7, R-L 59.5, and C 44.6. 

I trained the model with python3.6.10, pytorch0.3.1, and TitanXP gpu.

**Comment by ramakanth-pasunuru** [comment]
Hi, if you see the training details in the supplementary of the paper, you will notice that we used 5-average-ensemble models for reporting the main table results.

**Comment by tuyunbin** [comment]
Hi, I do not konw exactly what's the meaning of 5-average-ensemble.  Does that mean you selected top-5 best models on the validation set and computed a average score among them?
